src/documentation/components/Navigation.vue

```diff
@@ -35,6 +35,9 @@
             >
           </li>
           <li><NuxtLink to="/components/phone-input">Phone input</NuxtLink></li>
+          <li>
+            <NuxtLink to="/components/forms/radio-button">Radio button</NuxtLink>
```

Comment:
i'm getting a page not found error and it's probably because the navigation points to forms/radio-button but your radio-button folder is not inside forms/ 🤔